### PR TITLE
Clee/docs 240

### PR DESCRIPTION
**Pull request by clee-stytch**
Add SMS OTP authentication to the demo app. Tested manually for all new and existing functionalities.

### CODE REVIEW

**Review by edwin-stytch — APPROVED** [review]
@clee-stytch 

omg could it actually be... 

Screenshot 2024-02-27 at 7 06 35 PM
 
🪄 ✨ ship it

**Comment by chris-stytch** [comment]
One quick ask, while we're here, would you mind updating the SDK versions to latest? i.e. `stytch-node`, `@stytch/nextjs`, and `@stytch/vanilla-js`

File: src/components/Authenticate.tsx [hunk 1]
```diff
@@ -25,6 +25,8 @@ const Authenticate = () => {
     if (stytch && !user && isInitialized) {
       const token = searchParams.get("token");
       const stytch_token_type = searchParams.get("stytch_token_type");
+      const code = searchParams.get("code");
```
Comment (chris-stytch) [inline src/components/Authenticate.tsx:28]: Is this code necessary? 

An OTP flow (when using the pre-built UI) doesn't involve a redirect, and thus won't hit this route. The OTP code should be entered (and auth'd) all in the login component completely by the JS SDK.

**Review by chris-stytch — CHANGES_REQUESTED** [review]

File: src/components/Authenticate.tsx (hunk 1 above)
Comment (clee-stytch) [reply src/components/Authenticate.tsx]: Ah you're right, I've looked through the JS SDK and see that we can grab the user input as part of the state to pass to the authenticate call. Updated!

**Comment by clee-stytch** [comment]
> Couple quick asks:
> 
> * While we're here, would you mind updating the SDK versions to latest? i.e. `stytch-node`, `@stytch/nextjs`, and `@stytch/vanilla-js`
> 
> In order to send SMS messages, you must first add your credit card, this is to limit fraud/spam and in case there are incidental charges.

Done!

File: src/components/Authenticate.tsx (hunk 1 above)
Comment (chris-stytch) [reply src/components/Authenticate.tsx]: To put a finer point on it, no backend authenticate call is necessary for OTP when you're using our pre-built UI components. Unlike with EML or OAuth, no redirect happens as part of the authentication flow, so the user will never hit the redirect. 

So for adding OTP to this app, all we'll need to do is the work in `Login.tsx`, no other code changes should be necessary to get OTP up and running. Mind removing the changes here to `Authenticate.tsx` and giving it a test?

Comment (clee-stytch) [reply src/components/Authenticate.tsx]: Oh woah magic! Okay just updated - appreciate the clarification.

**Review by chris-stytch — APPROVED** [review]
Let's ship!

**Pull request merged**